setup/flex.rst

```diff
@@ -111,7 +111,7 @@ Symfony application by executing the following command:
 
 .. code-block:: terminal
 
-    $ composer create-project symfony/skeleton my-project
+    $ composer create-project symfony/skeleton my-project 4.1
```

Comment:
IMO we should use a constraint that will pick the latest patch release for 4.1 like this: `composer create-project symfony/skeleton:^4.1 my-project`